students/mariusz_michalczyk/lesson_07_strings/table_printer.py

```diff
@@ -3,22 +3,23 @@
               ['dogs', 'cats', 'moose', 'goose']]
 
 
-def get_biggest_len(col):
-    bgst_len = 0
-    for item in col:
-        if len(item) > bgst_len:
-            bgst_len = len(item)
-    return bgst_len
-
-
 def print_table(table_data):
-    col_width = len(table_data[0])
+    col_width, max_word_len = get_maxs(table_data)
```

Comment:
It's still not what they requested. In the example the longest word is "cherries"(8 letters), but looking at the second column, the words are right adjusted to fit the "Alice", "Carol" and "David" (5 letters), not "cherries". For that you would need a maximum for each column, not a global maximum for the whole table. And you don't need the `col_width`, `enumerate` will do.